cdk.json

```diff
@@ -33,7 +33,7 @@
 
     "atat:CspConfigurationName": "config/csp/integration",
     "atat:GitHubPatName": "auth/github/pat",
-    "atat:VersionControlBranch": "develop",
+    "atat:VersionControlBranch": "feature/cloudwatch-log-group-cissp-updated",
```

Comment:
Assuming this is for testing, once testing is done might want to update back to `develop`.

Reply:
Agree this needs a second look